rero_ils/modules/documents/extensions/add_mef_pid.py

```diff
@@ -34,8 +34,8 @@ def add_mef_pid(self, record):
         from rero_ils.modules.contributions.api import Contribution
         agents = record.get('subjects', []) +\
             record.get('subjects_imported', []) + \
-            [c['agent'] for c in
-                record.get('contribution', []) if c.get('agent')]
+            [c['entity'] for c in
+                record.get('contribution', []) if c.get('entity')]
```

Comment:
Perfectly correct but shorter to use `[c['entity'] for c in record.get('contribution', []) if 'entity' in c]`
(maybe it will take place on a single line)